jsapp/js/designSystem/textboxDemo.tsx

```diff
@@ -0,0 +1,224 @@
+import React from 'react'
+import bem from 'js/bem';
+import Checkbox from 'js/components/common/checkbox'
+import TextBox, {AvailableType} from 'js/components/common/textBox'
+import Radio from 'js/components/common/radio'
+
+const textBoxTypes: AvailableType[] = ['text-multiline', 'text', 'email', 'password', 'url', 'number']
+
+const errorsValues: {[val: string]: any} = {
+  no: null,
+  messageless: true,
+  one: 'You need to type something else!',
+  multi: [
+    "First error is about one thing!",
```

Comment:
Not a big deal at all, but if we clear the text box then both icons in start and end show up:
![Screenshot from 2022-01-03 16-05-26](https://user-images.githubusercontent.com/16762675/147980286-f5c7ff2e-c584-4b01-a6fb-154a51eac253.png)

maybe a niche request but could we make these editable? may be useful to see how a responseJSON would look

Reply:
That would take some more time. But I did update the comment next to `errors` property in `textBox.tsx` to make it clear what you need to pass to component.